packages/babel-template/src/index.js

```diff
@@ -10,7 +10,100 @@ import * as t from "babel-types";
 let FROM_TEMPLATE = "_fromTemplate"; //Symbol(); // todo: probably wont get copied over
 let TEMPLATE_SKIP = Symbol();
 
-export default function (code: string, opts?: Object): Function {
+export default function (firstArg, ...rest) {
+  if (typeof firstArg === "string") {
+    return factory(firstArg, ...rest);
+  } else {
+    return template(firstArg, ...rest);
+  }
+}
```

Comment:
I tried making a separate export instead, but that would be a breaking change because of `add-module-exports`.

Can we get rid of it? 🙂